packages/flutter/lib/src/services/keyboard_key.dart

```diff
@@ -0,0 +1,2543 @@
+  static const LogicalKeyboardKey launchKeyboardLayout = LogicalKeyboardKey(0x001000c01ae, debugName: kReleaseMode ? null : 'Launch Keyboard Layout');
+
+  /// Represents a logical "Launch Screen Saver" key on the keyboard. See the
+  /// function [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey launchScreenSaver = LogicalKeyboardKey(0x001000c01b1, debugName: kReleaseMode ? null : 'Launch Screen Saver');
+
+  /// Represents a logical "Launch Audio Browser" key on the keyboard. See the
+  /// function [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey launchAudioBrowser = LogicalKeyboardKey(0x001000c01b7, debugName: kReleaseMode ? null : 'Launch Audio Browser');
+
+  /// Represents a logical "New Key" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey newKey = LogicalKeyboardKey(0x001000c0201, debugName: kReleaseMode ? null : 'New Key');
+
+  /// Represents a logical "Close" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey close = LogicalKeyboardKey(0x001000c0203, debugName: kReleaseMode ? null : 'Close');
+
+  /// Represents a logical "Save" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey save = LogicalKeyboardKey(0x001000c0207, debugName: kReleaseMode ? null : 'Save');
+
+  /// Represents a logical "Print" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey print = LogicalKeyboardKey(0x001000c0208, debugName: kReleaseMode ? null : 'Print');
+
+  /// Represents a logical "Browser Search" key on the keyboard. See the
+  /// function [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey browserSearch = LogicalKeyboardKey(0x001000c0221, debugName: kReleaseMode ? null : 'Browser Search');
+
+  /// Represents a logical "Browser Home" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey browserHome = LogicalKeyboardKey(0x001000c0223, debugName: kReleaseMode ? null : 'Browser Home');
+
+  /// Represents a logical "Browser Back" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey browserBack = LogicalKeyboardKey(0x001000c0224, debugName: kReleaseMode ? null : 'Browser Back');
+
+  /// Represents a logical "Browser Forward" key on the keyboard. See the
+  /// function [RawKeyEvent.logicalKey] for more information.
+  static const LogicalKeyboardKey browserForward = LogicalKeyboardKey(0x001000c0225, debugName: kReleaseMode ? null : 'Browser Forward');
+
+  /// Represents a logical "Browser Stop" key on the keyboard. See the function
+  /// [RawKeyEvent.logicalKey] for more information.
+  static c
```

Comment:
This should mention that the PhysicalKeyboardKey is pretending that the user typed on a QWERTY keyboard.

Adding a concrete example of the difference between Physical and Logical would also be great (e.g. when you hit the Z letter on a QERTZ keyboard, logical is Z, physical is Y).